backend/src/users/controllers.ts

```diff
@@ -240,11 +241,23 @@ const getUsers = async (
     skip: skip,
     cursor: cursor,
   });
+
+  // Get hours for each user
+  const newQueryResult: any = queryResult;
+  for (let i = 0; i < queryResult.length; i++) {
+    const user = queryResult[i];
+    const hours = await getHours(user.id);
+    newQueryResult[i].totalHours = hours;
+  }
+
```

Comment:
I thought we only want to grab the hours if the query specifies?

Reply:
fixed